.github/actions/build_ci/entrypoint.sh

```diff
@@ -111,25 +94,24 @@ build_zephyr(){
 	pip3 install west || exit 1
 
 	export PROJECT_ROOT=$PWD
-	wget $ZEPHYR_SDK_DOWNLOAD_URL --progress=dot:giga || exit 1
-	echo "Extracting $ZEPHYR_SDK_TAR"
-	pv $ZEPHYR_SDK_TAR -i 3 -ptebr -f | tar xJ || exit 1
-	rm -rf $ZEPHYR_SDK_INSTALL_DIR || exit 1
-	yes | ./$ZEPHYR_SDK_SETUP_DIR/setup.sh || exit 1
 	west init --mr $ZEPHYR_VERSION ./zephyrproject || exit 1
 	cd ./zephyrproject || exit 1
-	west update || exit 1
+	west update -n || exit 1
 	west zephyr-export || exit 1
-	pip3 install  -r ./zephyr/scripts/requirements.txt || exit 1
+	west packages pip --install || exit 1
+
 	echo  "Update zephyr OpenAMP repos"
 	#Update zephyr OpenAMP repos
 	cp -r $PROJECT_ROOT/open-amp/lib modules/lib/open-amp/open-amp/ || exit 1
 	cp $PROJECT_ROOT/open-amp/CMakeLists.txt modules/lib/open-amp/open-amp/ || exit 1
 	cp $PROJECT_ROOT/open-amp/VERSION modules/lib/open-amp/open-amp/ || exit 1
 	cp -r $PROJECT_ROOT/open-amp/cmake modules/lib/open-amp/open-amp/ || exit 1
 	cp -r $PROJECT_ROOT/libmetal modules/hal/libmetal/ || exit 1
-	cd ./zephyr || exit 1
-	source zephyr-env.sh || exit 1
```

Comment:
I understrand the additions to install the sdk.
Are we sure we don't need to source the zephyr-env.sh any more?
Is this because west installed the sdk so knows where it is?